generators/app/templates/resource/tasks/config/eslint.js

```diff
@@ -0,0 +1,20 @@
+/**
+ * eslint
+ *
+ * ---------------------------------------------------------------
+ *
+ * # Lints js, jsx, and es6
+ *
+ * For usage docs see:
+ *      https://github.com/sindresorhus/grunt-eslint
+ */
+module.exports = function(grunt) {
+  grunt.config.set('eslint', {
+    target: ['src/*.js'],
```

Comment:
Shouldn't this include js? and sub folders? `src/**/*.jsx`?

Reply:
I added it but none of the other ship-components have that